cibyl/sources/elastic/client.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
We might want to change the directory name to elasticsearch since I think elastic is the name of the company or the whole stack (logstash, kibana) and this source is specifically about elasticsearch

Reply:
Yes, all the ELK stack. Changed.